pkg/proxy/proxy_test.go

```diff
@@ -595,9 +726,45 @@ type specData struct {
 
 func (s *specData) fakeSpecCreator(*http.Request, *zap.Logger) (apisLoadTestV1.LoadTestSpec, error) {
 	lt := apisLoadTestV1.LoadTestSpec{}
+
 	distributedPods := int32(s.distributedPods)
+
+	pairs := strings.Split(strings.TrimSpace(s.tags), ",")
+	tags := make(map[string]string, len(pairs))
+
+	for _, pair := range pairs {
+		pair = strings.TrimSpace(pair)
+		if len(pair) < 1 {
+			continue
+		}
+
+		parts := strings.SplitN(pair, ":", 2)
+		label := strings.TrimSpace(parts[0])
+
+		if len(label) == 0 {
+			continue
+		}
+
+		if len(parts) != 2 {
+			continue
+		}
+
+		value := strings.TrimSpace(parts[1])
+
+		if len(value) == 0 {
+			continue
+		}
+
+		if len(value) > maxTagLength {
+			continue
+		}
+
+		tags[label] = value
+	}
```

Comment:
Do we really need to duplicate this logic here? If we would extract https://github.com/hellofresh/kangal/pull/39/files#diff-0530307c704a9dc589d79d6e98bd73aaR85-R108 that func could just be called.

Reply:
This is a mock parser, the intention is to ignore the error because we don't need it. But we can return error, no problem with that. So I can merge with the one from https://github.com/hellofresh/kangal/pull/39#discussion_r502276632